tests/test_ignore.py

```diff
@@ -0,0 +1,169 @@
+from vulture import core
+
+from . import check, skip_if_not_has_async
+
+
+def check_ignore(code, ignore_names, ignore_decorators, expected):
+    v = core.Vulture(
+            verbose=True, ignore_names=ignore_names,
```

Comment:
Use 4 spaces to indent.

Reply:
Hey, my editor is already showing 4 space indentation.

Do you mean that we should be using the following format instead? 😅 
```python
def check_ignore(code, ignore_names, ignore_decorators, expected):
    v = core.Vulture(verbose=True, ignore_names=ignore_names,
                     ignore_decorators=ignore_decorators)
    v.scan(code)
    check(v.get_unused_code(), expected)
```

Reply:
Hmm, are you sure about this, because starting on column 13 on line9 makes flake8 angry:
```
tests/test_ignore.py:8:9: E128 continuation line under-indented for visual indent
```

Reply:
OK, let's leave it as it is then :-)